### ISSUE TITLE
Update Spock 1.3-groovy-2.5 to support Java 13

### ISSUE THREAD

**Issue by kdowbecki** [spockframework/spock#1051]
# Issue description
Spock 1.3-groovy-2.5 depends on Groovy 2.5.4 which does not work with Java 13. Upgrading to Groovy 2.5.7 or newer solves the problem. See [GROOVY-9020](https://issues.apache.org/jira/browse/GROOVY-9020).

# How to reproduce
Run tests with `spock-spring-1.3-groovy-2.5` or other `1.3-groovy-2.5` dependency on Java 13. This will throw:

```
Caused by: BUG! exception in phase 'semantic analysis' 
  in source unit '/home/myself/project/Test.groovy' 
  Unsupported class file major version 57
```

# Additional Environment information
OpenJDK 13.0.1 Zulu

## Groovy version
2.5.4

### Gradle
6.0

## Operating System
Linux

### Gradle/Grails
    testCompile group: 'org.spockframework', name: 'spock-spring', version: '1.3-groovy-2.5'

**Comment by BoukeNijhuis** [comment]
If you do not mind using a snapshot version and doing some extra config in the pom.xml you can use the following pom. This one works with Java 14 EA (and all versions below). I am not sure how to translate this to Gradle, but that is probably not so hard.

https://github.com/BoukeNijhuis/spock-with-new-java-versions/blob/master/pom.xml

I hope this helps.

**Comment by bsfowlie** [comment]
I have no problems running Spock 1.3-groovy-2.5 on Groovy 2.5.8, I just needed to put the dependency for Groovy 2.5.8 into my build.gradle file to override the version included by Spock 1.3-groovy-2.5. Should work the same if you are using Maven, just provide the Groovy dependency in the pom.xml file to override what comes with Spock.

I can provide a sample build.gradle file if needed.

**Comment by SteveHigham** [comment]
I would like to see a Spock version with Groovy above 2.5.4 as well. I am developing on a Mac and this causes multiple tests to fail with an IllegalAccess exception when I moved from JDK 1.8 to JDK 11. This was using 1.3-groovy-2.5.
This issue is fixed by using 2.0-M1-groovy-2.5 but this is not a release version and is causing additional problems which I will record separately.

**Comment by leonard84** [comment]
Thanks for raising the issue, but there won't be another 1.x release and since this can be easily handled by managing the groovy version as @bsfowlie mentioned above I'll close this.
